ui/src/Dashboard/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react'
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+// import loginSer from "./login.service";
+// import authSer from "../shared/service/auth";
+
+import Button from '@material-ui/core/Button';
+import FormControl from '@material-ui/core/FormControl';
+import { useHistory } from "react-router-dom";
+import Home from '../Home';
+
+//import './login.css';
+export default function Login(props: any): any {
+    let history = useHistory();
+    const [state, setState] = React.useState({
+        email: "",
+        pass: "",
+    });
+
+    const handleEmail = (e: any) => {
+        let val = e.target.value;
+        setState({ ...state, email: val });
+    };
+
+    const handlePass = (e: any) => {
+        let val = e.target.value;
+        setState({ ...state, pass: val });
+    };
+
+    const redirectDashbord = () => {
+        props.LoginFun();
+        let url = "/home"
```

Comment:
Can be done in one line. No need of declaration

Reply:
converted to one line